src/main/java/core/basesyntax/service/impl/ReportServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.ReportService;
+import java.util.Map;
+
+public class ReportServiceImpl implements ReportService {
+    private static final String REPORT_TITLE = "fruit,balance";
+    private static final String COMMA = ",";
+    private StorageDao fruitStorageDao = new StorageDaoImpl();
```

Comment:
let's create new StorageDaoImpl() in main and pass it to ReportServiceImpl constructor
or we can pass `Set<Map.Entry<Fruit, Integer>> entiries` as input parameter to the method makeReport()